sparkles/core.py

```diff
@@ -456,6 +456,12 @@ def review_status(self):
 
         return status
 
+    @property
+    def att_targ(self):
+        if not hasattr(self, '_att_targ'):
+            self._att_targ = self._calc_targ_from_aca(self.att, 0, 0)
```

Comment:
This looks good (computing the _att_targ as needed) though I suppose if we get the y_off z_off values there will be more places to scrub.  Also, not sure why _calc_targ_from_aca requires args at all, but that's not an issue for this PR.

Reply:
Agreed those should be optional kwargs.  When we get offsets in the code here it will be an easy scrub.